src/lazy/components/server.tsx

```diff
@@ -30,6 +33,11 @@ export function createComponentServer<C extends ComponentType<any>>({
     return (
       <>
         <input type="hidden" data-lazy-begin={dataLazyId} />
+        {defer !== PHASE.LAZY &&
+          SETTINGS.MANIFEST[moduleId] &&
+          SETTINGS.MANIFEST[moduleId].map(url => (
+            <link key={url} rel="preload" href={url} as="script" />
+          ))}
```

Comment:
Decided to render `<link preload />` tags only if not generic lazy component, as I'm worried too many things might be preloaded